src/main/resources/templates/macros/json/output.vm

```diff
@@ -1,16 +1,21 @@
-#macro(includeOutput, $output)
+#macro(includeOutput, $outputs, $isPassed)
 
-#if ($output)
+#if($outputs)
   <div class="outputs inner-level">
-    #foreach($message in $output.getMessages())
+    #foreach($output in $outputs)
```

Comment:
provided screenshtos don't really shows what is your intention. Can you describe it?